src/app/domain/geolocation/tracking/user.js

```diff
@@ -53,15 +61,19 @@ export const getOrCreateUser = async user => {
       body: JSON.stringify({ user: user })
     })
     if (respFetch.status === 403) {
+      Log("No openpath user found, let's create one.")
       return createUser(user)
     }
     response = await respFetch.json()
     const uuid = response?.user_id?.['$uuid']
     if (!uuid) {
+      Log("No openpath uuid found, let's create the user.")
       await createUser(user)
     }
     return uuid
   } catch (err) {
+    // If the user actually exist, the creation will return the existing user
```

Comment:
I'm not sure to understand this comment. Is this a mechanism on cozy-stack side?

Reply:
No it's related to openpath: when you request a user creation, with an already existing id, it will return the existing user, without creating a new one